Gemfile

```diff
@@ -50,6 +50,10 @@ gem 'chartkick', '~> 3.3', '>= 3.3.1'
 
 gem 'sidekiq-cron', '~> 1.1'
 
+gem 'rails_performance', '~> 0.9.1'
```

Comment:
can we also add rack mini profiler? I'd really like to learn about that tool 😬 https://github.com/MiniProfiler/rack-mini-profiler

Reply:
Sure!